cmd/ak/cmd/manifest/plan.go

```diff
@@ -0,0 +1,73 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmanifest"
+)
+
+var noValidate, fromScratch bool
+
+var planCmd = common.StandardCommand(&cobra.Command{
+	Use:     "plan [file] [--no-validate] [--from-scratch]",
+	Short:   "plan manifest application",
+	Aliases: []string{"p"},
+	Args:    cobra.MaximumNArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		data, path, err := common.Consume(args)
+		if err != nil {
+			return err
+		}
+
+		manifest, err := sdkmanifest.Read(data, path)
+		if err != nil {
+			return err
+		}
+
+		ctx, cancel := common.LimitedContext()
+		defer cancel()
+
+		actions, err := sdkmanifest.Plan(
+			ctx,
+			manifest,
+			common.Client(),
+			sdkmanifest.WithLogger(func(msg string) { fmt.Println(msg) }),
+			sdkmanifest.WithFromScratch(fromScratch),
+		)
+		if err != nil {
+			return err
+		}
+
+		common.Render(actions)
+
+		return nil
+	},
+})
+
+func init() {
+	planCmd.Flags().BoolVarP(&noValidate, "no-validate", "n", false, "do not validate")
+	planCmd.Flags().BoolVarP(&fromScratch, "from-scratch", "s", false, "assume no existing setup")
+}
+
+func plan(data []byte, path string) (sdkmanifest.Actions, error) {
```

Comment:
Did you mean to call this function in this file too?

Reply:
yes, done.